### PR TITLE
Project initialization and Setup

### PR DESCRIPTION
**Pull request by ericmahare**
### In this section I  did the following : -

- [ ] Initialized React app.
- [ ] Created the required components
- [ ] Add Books and Categories routes in the navigation bar

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <article className='book'>
+      <div>
+        <h5>Action</h5>
+        <h3>The Hunger Games</h3>
+        <p className="book-info">Suzanne Collins</p>
+        <div className='article-footer'>
+          <span>Comments |</span>
+          <span>Remove |</span>
+          <span>Edit |</span>
```
Comment (Cathella) [inline src/components/Book.js]: - [ ] [REQUIRED] Kindly make these buttons especially the delete button as mentioned in the requirements

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const Book = () => {
+  return (
+    <article className='book'>
+      <div>
+        <h5>Action</h5>
+        <h3>The Hunger Games</h3>
+        <p className="book-info">Suzanne Collins</p>
```
Comment (Cathella) [inline src/components/Book.js]: - [ ] [REQUIRED] Kindly make sure that this component takes the book data as a prop and renders the title and the author. The title and author should be dynamic not static

File: src/components/Form.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  return (
+    <div className="form-section">
+    <h4>ADD NEW BOOK</h4>
+    <form>
+      <input type='text' placeholder='book title'/>
+      <select id="cars" name="cars">
+        <option value="volvo">Author</option>
+      </select>
+      <button className="form-btn">Add book</button>
+    </form>
```
Comment (Cathella) [inline src/components/Form.js]: ✅ Renders a form with inputs for the title 👍🏻 
✅ Has a button to submit 👍🏻 

- [ ] [REQUIRED] Kindly add an input for the author of the book

File: src/components/Categories.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const Categories = () => {
+  return (
+    <button className="cat-btn">Check status</button>
```
Comment (Cathella) [inline src/components/Categories.js:3]: ✅ Should display only a button with text "Check status" 👍🏻

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @ericmahare 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct GitHub Flow.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

this-close-so-close


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
Hi @ericmahare , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**